geomstats/geometry/pullback_metric.py

```diff
@@ -351,26 +209,26 @@ def diffeomorphism(self, base_point):
             Inner-product matrix.
         """
 
-    def raw_jacobian_diffeomorphism(self, base_point):
-        r"""Raw jacobian of the diffeomorphism.
+    @abc.abstractmethod
+    def inverse_diffeomorphism(self, image_point):
+        r"""Inverse diffeomorphism at base point.
+
+        Let :math:`f` be the diffeomorphism
+        :math:`f: M \rightarrow N` of the manifold
+        :math:`M` into the manifold `N`.
 
-        Raw jacobian autodiff of diffeomorphism regardless of vectorization.
+        :math:`f^-1: N \rightarrow M` of the manifold
 
         Parameters
         ----------
-        base_point : array-like, shape=[..., *shape]
+        image_point : array-like, shape=[..., *i_shape]
```

Comment:
I wonder if "image_point" is the more intuitive wording here..

output space // output point?
embedding space // embedded point?
immersed point?

Reply:
I have no preference here. I've simply used the terminology already present in the docstrings.